dcicutils/s3_utils.py

```diff
@@ -11,7 +11,7 @@
 ###########################
 # Config
 ###########################
-s3 = boto3.client('s3')
+s3 = boto3.client('s3', region_name='us-east-1')
```

Comment:
Alex had a branch, that for whatever reason didn't get merged in, but moved the s3 here to s3Utils.s3.  I think that needs to happen, maybe just implement it in the branch since Alex's branch has diverged.

Reply:
Done